server/src/infra/auth_service.rs

```diff
@@ -1,14 +1,8 @@
-use crate::{
-    domain::{
-        error::DomainError,
-        handler::{BackendHandler, BindRequest, GroupIdAndName, LoginHandler},
-        opaque_handler::OpaqueHandler,
-    },
-    infra::{
-        tcp_backend_handler::*,
-        tcp_server::{error_to_http_response, AppState},
-    },
-};
+use std::collections::{hash_map::DefaultHasher, HashSet};
```

Comment:
Is that cargo fmt reordering the imports? Maybe we should fuse the various std imports. Small nit, feel free to ignore.

Reply:
Not sure, I just formatted the file using rustfmt